pkg/operator/ceph/config/monstore.go

```diff
@@ -192,3 +192,18 @@ func (m *MonStore) DeleteAll(options ...Option) error {
 	}
 	return nil
 }
+
+// SetKeyValue sets an arbitrary key/value pair in Ceph's general purpose (as opposed to
+// configuration-specific) key/value store. Keys and values can be any arbitrary string including
+// spaces, underscores, dashes, and slashes.
+// See: https://docs.ceph.com/en/quincy/man/8/ceph/#config-key
+func (m *MonStore) SetKeyValue(key, value string) error {
+	logger.Debugf("setting %q=%q option to the general-purpose mon k/v store", key, value)
```

Comment:
is there a different mon store, or how about this?
```suggestion
	logger.Debugf("setting %q=%q option in the mon store", key, value)
```